storage/postgres/postgres_adapter.go

```diff
@@ -68,7 +68,7 @@ func WithTableName(tableName string) PgOptionFunc {
 // WithSslOn is an optional function to make ssl enabled; default is disabled
 func WithSslOn() PgOptionFunc {
 	return func(options *PgOptions) {
-		options.sslMode = "enable"
+		options.sslMode = "require"
```

Comment:
Are the certificate management steps required to use SSL actually supported here? I don't see it tested.

If not, it might be better to simply remove this option func.